xtras/src/address_map.rs

```diff
@@ -11,6 +11,22 @@ pub struct AddressMap<K, A> {
     inner: HashMap<K, Address<A>>,
 }
 
+/// A loud trait that makes sure we don't forget to return `StopAll` from the `stopping` lifecycle
+/// callback.
+///
+/// Things might be outdated when you are reading this so bear that in mind.
+/// There is an open patch to `xtra` that changes the default implementation of an actor's
```

Comment:
would you mind linking the open patch here?

Reply:
It is here: https://github.com/Restioson/xtra/pull/58

Might send a follow-up patch.

Reply:
Don't want to block the release now by adding more commits!

Reply:
> It is here: [Restioson/xtra#58](https://github.com/Restioson/xtra/pull/58)
> 
> Might send a follow-up patch.

that would be great, otherwise this comment has a high risk of becoming stale soon :)